kolibri/plugins/coach/assets/src/views/reports/channel-list-page.vue

```diff
@@ -62,17 +65,20 @@
     name: 'coachRecentPageChannelList',
     components: {
       reportTable,
-      reportSubheading,
       headerCell,
       nameCell,
       activityCell,
     },
     mixins: [alignMixin],
     $trs: {
       recentTitle: 'Recent activity',
+      topicsTitle: 'Content',
       channels: 'Channels',
       channelList: 'Channel list',
       lastActivity: 'Last active',
+      showingRecent: 'Showing activity in past {threshold} days',
```

Comment:
Will the `threshold` number be localized correctly this way?

Reply:
I .. think so. But will check

Reply:
These are new strings, so there's no way to check manually right now. Just not sure if you need some extra configuration to make sure it doesn't use arabic numerals by default.